index.js

```diff
@@ -0,0 +1,74 @@
+const subjects = [
+  "Programming",
+  "Data Science",
+  "Technology",
+  "Self Improvement",
+  "Self Improvement",
+  "Writing",
+  "Relationships",
+  "Machine Learning",
+  "Productivity",
+  "Politics",
+];
+
+async function getArticles(isTrending) {
+  const response = isTrending
+    ? await fetch("./jsons/trending-articles.json")
+    : await fetch("./jsons/articles.json");
+  const json = await response.json();
+  return json;
+}
+
+getArticles(true).then((trendingArticles) => {
+  const trends = document.createElement("div");
+  trends.classList.add("trending-articles");
+  for (let i = 0; i < 6; i++) {
```

Comment:
suggestion: change this `for` to `map` to make your code dynamic, beside that it's gonna be easy to get elements, clean and a bit short